packages-web/fieldset/src/components/Fieldset.tsx

```diff
@@ -0,0 +1,24 @@
+import { createElement, CSSProperties, FunctionComponent } from "react";
+
+export interface FieldsetProps {
+    name?: string;
+    className: string;
+    style?: CSSProperties;
+    tabIndex?: number;
+    legend?: string;
+}
+
+export const Fieldset: FunctionComponent<FieldsetProps> = (props): JSX.Element | null => {
```

Comment:
`export const Fieldset = (props: FieldsetProps): JSX.Element | null => {` works as well

Reply:
Prefer use Function instead of const. Agreed with @mijay

Reply:
Changed all components to functions. For this specific component I used the type `PropsWithChildren<FieldsetProps>` for the `props` parameter, so that the children prop can be used.

Reply:
Fine, valid case